xapian-letor/m4-macros/xapianletor.m4

```diff
@@ -0,0 +1,150 @@
+# Get XAPIANLETOR_CXXFLAGS, XAPIANLETOR_LIBS, and XAPIANLETOR_VERSION from xapianletor-config and
+# AC_SUBST() them.
+
+# serial 17
+
+# AC_PROVIDE_IFELSE(MACRO-NAME, IF-PROVIDED, IF-NOT-PROVIDED)
+# -----------------------------------------------------------
+# If this macro is not defined by Autoconf, define it here.
+m4_ifdef([AC_PROVIDE_IFELSE],
+	 [],
+	 [m4_define([AC_PROVIDE_IFELSE],
+		 [m4_ifdef([AC_PROVIDE_$1],
+			   [$2], [$3])])])
+
+# XO_LIB_XAPIANLETOR([ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND[ ,XAPIANLETOR-CONFIG]]])
+# --------------------------------------------------------
+# AC_SUBST-s XAPIANLETOR_CXXFLAGS, XAPIANLETOR_LIBS, and XAPIANLETOR_VERSION for use in
+# Makefile.am
+#
+# If ACTION-IF-FOUND and ACTION-IF-NOT-FOUND are both unset, then an
+# appropriate AC_MSG_ERROR is used as a default ACTION-IF-NOT-FOUND.
+# This allows XO_LIB_XAPIANLETOR to be used without any arguments in the
+# common case where Xapian is a requirement (rather than optional).
+#
+# XAPIANLETOR-CONFIG provides the default name for the xapianletor-config script
+# (which the user can override with "./configure XAPIANLETOR_CONFIG=/path/to/it").
+# If unset, the default is xapianletor-config.  Support for this third parameter
+# was added in Xapian 1.1.3.
```

Comment:
This last sentence isn't true of `XO_LIB_XAPIANLETOR`.

Reply:
fixed in a fixup commit.